### ISSUE TITLE
Open grid with tons of images on avatar & pokemon click

### ISSUE THREAD

**Issue by coyotte508** [po-devs/po-web#91]
It'd be cool, if you clicked the avatar and you just had an overlay with all the possible avatars layed out in a grid-like manner. And you click it and it sets to it.

Same with pokemon.

No?

**Comment by lamperi** [comment]
True, the selector using arrows and arbirary number is difficult for new users.
